app/controllers/api/cases_controller.rb

```diff
@@ -0,0 +1,51 @@
+class API::CasesController < ApplicationController
+  respond_to :json
+
+  def create
+    _case = Case.new(case_params)
+    begin
+      saved = _case.save!
+    rescue ActiveRecord::StatementInvalid => invalid
+      return render json: {message: 'Invalid case'}
+    end
+    if saved
+      return render json: {message: 'Case successfully created!',
+                           case: _case}
+    else
+      return render json: {error: _case.errors.full_messages,
+                           status: 500}
+    end
+    return render json: {message: params[:type_of_case]}
+  end
+
+  def case_params
+    params.permit(
+      :id,
+      :user_id,
+      :type_of_case,
+      :pro_bono_placement,
+      :grant,
+      :initial_invoice_date,
+      :last_invoice_date,
+      :date_rec_initial_disbursement,
+      :date_rec_last_disbursement,
+      :case_tracking,
+      :date_mta_filed,
+      :case_supervisor_id,
+      :program,
+      :legal_case_name,
+      :judge_assigned,
+      :trial_attorney,
+      :case_progress,
+      :date_biometrics_done,
+      :lodge_or_rn_date,
+      :date_mta_filed,
+      :asylum_officer,
+      :nexus_argued,
+      :nexus_granted,
+      :case_outcome,
+      :case_outcome_achieved,
+      :date_of_outcome
+    )
+  end
```

Comment:
- [ ] populate API controller with `show`, `delete`, `update`, etc. (see other API controllers for reference)